app/build.gradle

```diff
@@ -23,15 +24,17 @@ android {
         }
     }
 }
+
 repositories {
     maven { url "${System.env.RIDEMETRIC_LOCAL_REPO_HOME}/ridemetricPublicRepo/repository" }
+    maven { url "${System.env.RIDEMETRIC_LOCAL_REPO_HOME}/ridemetricPrivateRepo/repository" }
```

Comment:
It's better to remove maven { url "${System.env.RIDEMETRIC_LOCAL_REPO_HOME}/ridemetricPublicRepo/repository" } also